### DISCUSSION TITLE
Reconsider `wordPosition` ranking rule

### DISCUSSION THREAD

**Discussion by curquiza** [meilisearch/product#329] [Feedback & Feature Proposal]
Before v0.21.0 we had two criterion
- `attribute` that made the content of the first attributes more relevant than the last one
- `wordPosition` that made the content at the beginning of the attribute more relevant than the end

In v0.21.0 and after we merged both in `attribute` criterion which does the same behavior as the old `wordPosition` and `attribute` criteria.

This post is to reconsider the separation between both rules. I'm not personally against or pro, I just open the post.

Here are some user feedback:
- https://github.com/meilisearch/MeiliSearch/issues/1770#issuecomment-981838643

**Comment by gmourier** [comment]
Locking this since we are releasing a stable v1.0 version.

Please open up a new discussion if it's needed 🙇‍♂️
